src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/IntegrationTestBase.kt

```diff
@@ -124,14 +124,9 @@ abstract class IntegrationTestBase {
         registry.add("spring.flyway.user", pgContainer::getUsername)
         registry.add("spring.flyway.password", pgContainer::getPassword)
       }
-
       lsContainer?.run {
-        lsContainer.getEndpointConfiguration(org.testcontainers.containers.localstack.LocalStackContainer.Service.SNS)
-          .let { it.serviceEndpoint to it.signingRegion }
```

Comment:
we are not setting it.serviceEndpoint to it.signingRegion anymore. is that fine?